Resources/public/css/theme/views/editpreview.css

```diff
@@ -5,34 +5,61 @@
 
 .ez-view-editpreviewview {
     background: #f3f3f3;
+    color: #555;
 }
 
-.ez-preview-content .ez-preview-hide {
-    color: #555;
+.ez-view-editpreviewview .ez-preview-iframe-wrapper:before {
+    transition: opacity ease 0.3s;
+    opacity: 0;
+
```

Comment:
yep on purpose, just to show/hide the loading animation with a nice transition on the opacity.